### DISCUSSION TITLE
Adetailer settings presets

### DISCUSSION THREAD

**Discussion by Woisek** [Bing-su/adetailer#530] [Ideas]
During my work and depending on the topics I generate, I see me often change settings in Adetailer, mainly the prompts. SO, I use different prompts for faces, portraits, landscapes, etc. so I have to constantly change settings.
And eve inf setting are saved in the PNGs itself, it never restores the Adetailer settings used for that.

So I thought it would be very useful to have the possibility to save presets for certain tasks, so we can quickly change the settings used for the current generating images. That would be really great. 🙂

**Comment by ride5k** [comment]
this would be a big time-saver

**Comment by uorufu** [comment]
This can be done with the ["config presets" ](https://github.com/Zyin055/Config-Presets) extension

you need to change the tracked components files to add the adetailer fields

here are my tracked components txt files. Put these in config preset directory and you should be good to go using up to 10 different adetailer tabs.
[config-img2img-custom-tracked-components.txt](https://github.com/Bing-su/adetailer/files/14797364/config-img2img-custom-tracked-components.txt)
[config-txt2img-custom-tracked-components.txt](https://github.com/Bing-su/adetailer/files/14797365/config-txt2img-custom-tracked-components.txt)

**Reply by Woisek** [reply]
Thanks for pointing this out. There once was such an extension (maybe it exists still), but it was a bit cumbersome and at some point broke, because of constant name changes of the elements to save (or so it was). 
So, I rather would like to see a 'build in save' in Adetailer and not _another_ extension to load. 🙂
Thanks nevertheless.
